app/controllers/merchant/items_controller.rb

```diff
@@ -1,5 +1,16 @@
 # frozen_string_literal: true
 
 class Merchant::ItemsController < Merchant::BaseController
-  def index; end
+  def index
+    @items = current_user.merchant.items
+  end
+
+  def activate_deactivate
+    if params[:activate_deactivate] == 'deactivate'
+      item = Item.find(params[:item_id])
+      item.toggle!(:active?)
+      flash[:notice] = "#{item.name} is now deactivated and is no longer for sale."
+      redirect_to merchant_user_items_path
+    end
+  end
```

Comment:
So does this only deactivate an item? I see the query params, but it only shows for deactivate.

Reply:
In the next user story (#59), a merchant admin should be able to activate an item. The intention was to complete both of these stories consecutively, but Laura were stuck for a long time trying to determine why our test wasn't passing, even though functionality appeared to be working as expected.